utils/useUpdatePreviewRef.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import Cookies from 'js-cookie'
+import { repoName } from 'prismic-configuration'
+
+function useGetExitPreviewRoute() {
```

Comment:
Change this function name to `getExitPreviewRoute` because we're going to change this from a hook to just a helper function.